src/QMCDrivers/DMC/DMCBatched.cpp

```diff
@@ -429,6 +432,14 @@ bool DMCBatched::run()
   IndexType num_blocks = qmcdriver_input_.get_max_blocks();
 
   estimator_manager_->startDriverRun();
+
+  //start walker log manager
+  wlog_manager_ = std::make_unique<WalkerLogManager>(walker_logs_input, allow_walker_logs, get_root_name(), myComm);
```

Comment:
The idea of the log manager being a run scope object is promising but then it should not be a member variable.

Reply:
Good point. We may get a chance to make it a run scope object after restructuring certain bits. Will give it a try once #5039 got merged.